frontend/src/components/place/AddPlace.vue

```diff
@@ -445,63 +470,58 @@ export default {
+                        end:    new Date(0,0,0,21)
+                    },
+                    'tu': {
+                        start:  new Date(0,0,0,10),
+                        end:    new Date(0,0,0,21)
+                    },
+                    'we': {
+                        start:  new Date(0,0,0,10),
+                        end:    new Date(0,0,0,21)
+                    },
+                    'th': {
+                        start:  new Date(0,0,0,10),
+                        end:    new Date(0,0,0,21)
+                    },
+                    'fr': {
+                        start:  new Date(0,0,0,10),
+                        end:    new Date(0,0,0,21)
+                    },
+                    'sa': {
+                        start:  new Date(0,0,0,10),
+                        end:    new Date(0,0,0,21)
+                    },
+                    'su': {
+                        start:  new Date(0,0,0,10),
+                        end:    new Date(0,0,0,21)
+                    }
+                }
             },
-            categories: {},
+            allCategories: [],
+            allFeatures: [],
             selectedTag: '',
             category_tags: [],
+
             weekdays: [],
-            timeStart: new Date(),
-            timeEnd: new Date(),
-            features: [
-                {
-                    name: 'wi-fi',
-                    status: false
-                },
-                {
-                    name: 'hookah',
-                    status: false
-                },
-                {
-                    name: 'music',
-                    status: false
-                },
-                {
-                    name: 'credit cards',
-                    status: false
-                },
-                {
-                    name: 'wheelchair accessible',
-                    status: false
-                },
-                {
-                    name: 'reservations',
-                    status: false
-                },
-                {
-                    name: 'parking',
-                    status: false
-                },
-                {
-                    name: 'restroom',
-                    status: false
-                },
-                {
-                    name: 'take-out',
-                    status: false
-                },
-                {
-                    name: 'delivery',
-                    status: false
-                }
-            ]
+            timeStart: new Date(0,0,0,10),
+            timeEnd: new Date(0,0,0,21)
         };
     },
 
     created() {
         this.$store.dispatch('category/getAllCategories')
             .then((result) => {
-                this.categories = result;
+                this.allCategories = result;
+            });
+
+        this.$store.dispatch('features/getAllFeatures')
+            .then((result) => {
+                this.allFeatures = result;
```

Comment:
You can use features:[] state from features module

Reply:
Done.